src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto

```diff
@@ -150,6 +150,9 @@ message ImpExt {
     // Supported viewability vendors. For list of supported vendors, visit:
     // https://resources.rubiconproject.com/resource/publisher-resources/xapi-specifications/#4.9.
     repeated string viewabilityvendors = 3;
+
+    // Ad unit code.
```

Comment:
Please update the comment to something like: Global Placement ID, as defined by TheTradeDesk. The goal is that all participants in buying impressions for the same placement get the same identifier, and that the identifier (as much as possible) is unique for the placement. (In general, this field will be populated with one of the DFP Ad Unit Code, Prebid Ad Slot, or some value provided by the publisher or ad server.)